crates/js-sys/CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ Released YYYY-MM-DD.
 
 ### Added
 
-* TODO (or remove section if none)
+* Added a re-export of `wasm-bindgen`.
```

Comment:
If you rebase onto `main` you will find a good template to do this.
The text is fine, just missing the link to the PR.